ckanext/resource_visibility/logic/action.py

```diff
@@ -0,0 +1,69 @@
+import ckan.plugins.toolkit as tk
+
+import ckanext.resource_visibility.constants as c
```

Comment:
This is a potentially confusing variable name, as it's common practice in CKAN to import `toolkit.c`.